web/app_options.js

```diff
@@ -310,6 +310,11 @@ const defaultOptions = {
     value: "",
     kind: OptionKind.API,
   },
+  enableHWA: {
+    /** @type {boolean} */
+    value: false,
+    kind: OptionKind.API + OptionKind.PREFERENCE + OptionKind.VIEWER,
```

Comment:
Nit: Let's keep the preference kind last for overall consistent.
```suggestion
    kind: OptionKind.API + OptionKind.VIEWER + OptionKind.PREFERENCE,
```